### PR TITLE
Update Readme to include installation & configuration steps

### PR DESCRIPTION
**Pull request by IPWright83**
Also bumps the version as I think it's good practise when anything external facing changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+        'app-copy/prefer-quot': 'error',
+    }
+}
+```
+
+## Rules
+
+### no-apologies
+
+This rule discourages the use of the word "sorry" in messages. Often it is included when in user facing messages when something goes wrong, but often it is not something we have any control over, so at Geckoboard we discourage it's use.
```
Comment (leocassarani) [inline README.md]:
```suggestion
This rule discourages the use of the word "sorry" in messages. Although it's sometimes included in user-facing error messages, most of the time we don't have any control over the cause of the error, so at Geckoboard we discourage its use.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+
+### prefer-quot
+
+This rule encourages the use of &quot (’) over &apos (') in text. This rule is **auto fixable**.
+
+#### valid
+```
+const MyComponent = () => (<span>That’s Leo’s mug!</span>);
+const stringLiteral = "You don’t have permission to do that!";
+const templateString = `It is ${name}’s dog`;
```
Comment (leocassarani) [inline README.md]:
```suggestion
const templateString = `Fido is ${name}’s dog`;
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,58 @@
...
+```
+
+#### invalid
+```
+const MyComponent = () => (<span>That's Leo's mug!</span>);
+const stringLiteral = "You don't have permission to do that!";
+const templateString = `It is ${name}'s dog`;
```
Comment (leocassarani) [inline README.md]:
```suggestion
const templateString = `Fido is ${name}'s dog`;
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,58 @@
...
+
+## Installation
+
+Assuming you already have ESLint installed in your project, simply run:
+
+`npm install --save-dev https://github.com/geckoboard/eslint-plugin-app-copy@latest`
+
+## Configuration
+
+Within your `.eslintrc` file you'll need to add this plugin to the plugins section, and add the additional rules:
```
Comment (leocassarani) [inline README.md:13]: Do we also want to include a set of instructions for ESLint 9, which [uses `eslint.config.js`](https://eslint.org/docs/latest/use/migrate-to-9.0.0#flat-config)?

**Comment by glenjamin** [comment]
Will this be used outside polecat?

If not I think we should see if we can embed it into the polecat repo, so we don't have to mess around with version numbers and publishing

If it needs to live separate, then I think we should adopt automatic version numbering and publishing

**Comment by leocassarani** [comment]
@glenjamin We weren't going to worry about publishing btw, you can install npm packages from GitHub so we were planning to do just that (`npm install https://github.com/geckoboard/eslint-plugin-app-copy@latest`).

**Comment by glenjamin** [comment]
> @glenjamin We weren't going to worry about publishing btw, you can install npm packages from GitHub so we were planning to do just that (`npm install https://github.com/geckoboard/eslint-plugin-app-copy@latest`).

Fair enough, that's what we do for `js-honeycomb-support` and it works well enough as long as the repo remains public

File: README.md [hunk 5]
```diff
@@ -1,3 +1,58 @@
...
+const MyComponent = () => (<span>That’s Leo’s mug!</span>);
+const stringLiteral = "You don’t have permission to do that!";
+const templateString = `Fido is ${name}’s dog`;
+```
+
+#### invalid
+```
+const MyComponent = () => (<span>That's Leo's mug!</span>);
+const stringLiteral = "You don't have permission to do that!";
+const templateString = `Fido is ${name}'s dog`;
```
Comment (leocassarani) [inline README.md]: Hang on, will this example actually work? I thought we couldn't support this because of the `\w'\w` regex.

Comment (IPWright83) [reply README.md]: Ooh yeah 🤦🏻‍♂️

Comment (IPWright83) [reply README.md]: Updated :)

**Review by leocassarani — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (IPWright83) [reply README.md:13]: I don't think we need to at this point, maybe when we test with eslint 9

**Pull request merged**